### PR TITLE
fix: Jump Table UsizeConversion Error

### PR DESCRIPTION
**Pull request by abigger87**
## Overview

Packed Jump Tables are being incorrectly generated.

Bytes were being correctly converted to a hexadecimal string using the utilities `bytes32_to_string` function, but then parsed into a usize using `.parse::<usize>()`. By default, this parse impl expects a base 10 decimal string.

The fix is to use `from_str_radix` which is implemented for rust's `usize`. Thus we can do:
```rust
usize::from_str_radix(bytes32_to_string(&jt.size, false).as_str(), 16)
```
where `jt.size` is the bytes32 jump table 32 bytes.

### CODE REVIEW

File: huff_codegen/src/lib.rs [hunk 1]
```diff
@@ -135,13 +137,16 @@ impl Codegen {
 
         res.utilized_tables.iter().try_for_each(|jt| {
             table_offsets.insert(jt.name.to_string(), table_offset);
-            let size = match bytes32_to_string(&jt.size, false).as_str().parse::<usize>() {
+            let size = match usize::from_str_radix(bytes32_to_string(&jt.size, false).as_str(), 16) {
```
Comment (clabby) [inline huff_codegen/src/lib.rs:140]: Maybe use your new `hex_to_usize` helper here?

**Review by clabby — CHANGES_REQUESTED** [review]
One small nit- otherwise looks great :smile:

File: huff_codegen/src/lib.rs (hunk 1 above)
Comment (abigger87) [reply huff_codegen/src/lib.rs:140]: ye defined that after :P will do now

**Review by clabby — APPROVED** [review]
LGTM!

**Pull request merged**